classes/logic.rb

```diff
@@ -0,0 +1,47 @@
+require 'json'
+require_relative './book'
+require_relative './label'
+
+class Logic
+  def add_label(item)
+    puts 'Label title: '
+    title = gets.chomp
+    puts 'Label color: '
+    color = gets.chomp
+    label = Label.new(title, color)
+    label.add_item(item)
```

Comment:
if you chose to change the way you handle your add label 
you may have to change some lines here 
it's up to you this since the app dose work

Reply:
If everything is working as per requirement, the listed changes are and should be optional, not required. kindly update your review!